plugins/katello/developers.md

```diff
@@ -229,37 +229,99 @@ Make sure to follow the instructions and update the docs to use the current rele
 
 Congratulations, you have successfully branched all of the repositories and prepared them for release. We recommend double checking each repository ensuring that the branch exists in the upstream repository, that the top of the commit stack in that branch includes the tag you created prior to branching (so that you have an initial tag point that represents the branch point) and that the tag itself exists. The last part of this step is to send to a follow up email letting the community know that everything has been branched.
 
-### Step 6. Pin katello-installer puppet modules for release
+### Step 6. Release puppet modules
 
-On the katello-installer's KATELLO-X.Y branch, we need to pin the modules to a minor release.
+Review the modules for releasing - we should be keeping them up to date when making incompatible changes, but do any need bumps of the major or minor version? If so, do so now.
 
-First, we take the pinned Foreman modules in the Puppetfile from the most current release, e.g. https://github.com/theforeman/foreman-installer/blob/1.9-stable/Puppetfile.  Overwrite the Foreman section in katello-installer's Puppetfile with these locked down versions.
+Go through the [list of managed modules](https://github.com/theforeman/foreman-installer-modulesync/blob/master/managed_modules.yml).
 
-Next, we'll lock down the Katello modules. First, Check out all the puppet modules:
+```sh
+# Review the the changes since the last tag
+tig $(git describe --abbrev=0 --tags)..HEAD
 
-```bash
-for module in candlepin certs capsule common elasticsearch gutterball katello crane service_wait pulp
-do
-  git clone git@github.com:Katello/puppet-${module}.git
-done
-```
+# If it needs a major or minor version bump
+# Note: usually we already bumped it for patch releases but do check!
+bundle exec rake module:bump:major
+bundle exec rake module:bump:minor
 
-Review the modules for releasing - we should be keeping them up to date when making incompatible changes, but do any need bumps of the major or minor version? If so, do so now.
+# Update CHANGELOG.md
+# TODO: modulesync to include github_changelog_generator in Gemfile
+# See https://github.com/skywinder/Github-Changelog-Generator#github-token on using a token to avoid ratelimits
+github_changelog_generator -u theforeman -p $(basename $PWD) --unreleased-only -o new-changelog.md --future-release $(bundle exec rake module:version)
+
+# Now merge new-changelog.md into CHANGELOG.md - do note that it doesn't handle
```

Comment:
https://github.com/skywinder/github-changelog-generator/issues/497

Reply:
I feel like this shouldn't be too big an issue for us, but I might be missing something.

Reply:
It it is - all our -stable branches and releases in them break github-changelog-generator. It assumes that anything merged since 3.1.0 was released is for 3.1.1, even if in the mean time we did a 2.1.1 release where we merged something.